### ISSUE TITLE
Variable importance/feature selection

### ISSUE THREAD

**Issue by sophiatinh** [yzontov/pls-da#62]
May I ask if there's an option to calculate variable importance (for eg. Variable Importance in Projection (VIP) scores) using this toolbox?

**Comment by yzontov** [comment]
No, the PLS-DA Tool does not have built-in option to calculate variable importance.
